lib/screens/explore_screens/regions_screens/region_event_details/region_event_details_screen.dart

```diff
@@ -20,162 +24,173 @@ class RegionEventDetailsScreen extends StatelessWidget {
     final RegionEventModel event = ModalRoute.of(context)!.settings.arguments! as RegionEventModel;
     final double width = MediaQuery.of(context).size.width;
     return BlocProvider(
-      create: (_) => RegionEventDetailsBloc(),
+      create: (_) => RegionEventDetailsBloc(event),
       child: BlocConsumer<RegionEventDetailsBloc, RegionEventDetailsState>(
```

Comment:
@gguij004 I really just need to redraw the bottom button so if you will not do rebuilds elsewhere I recommend this is a BlocListener and not a BlocConsumer